File: 2024-08-16/counter/count.py
import time

def count_to_100_million():
    start = time.time()
    res = 0

    for i in range(1, 100_000_001):
        res += 1

    duration = time.time() - start
    print(f"Counting to 100,000,000 took {duration:.6f} seconds")

def main():
    count_to_100_million()

if __name__ == "__main__":
    main()

File: 2024-08-16/counter/count.go
package main

import (
	"fmt"
	"time"
)

func main() {
	countTo100Million()
}

func countTo100Million() {
	start := time.Now()
	var res int

	for i := 1; i <= 6_000_000_000; i++ {
		res += 1
	}

	duration := time.Since(start)
	fmt.Printf("Counting to 100,000,000 took %v\n", duration)
}